addons/account/wizard/setup_wizards.py

```diff
@@ -82,59 +95,31 @@ def _compute_related_acc_type(self):
     def _number_unlinked_journal(self):
         return self.env['account.journal'].search([('type', '=', 'bank'), ('bank_account_id', '=', False)], count=True)
 
-    @api.onchange('linked_journal_id')
-    def _onchange_new_journal_code(self):
-        for record in self:
-            if not record.linked_journal_id:
-                record.new_journal_code = self.env['account.journal'].get_next_bank_cash_default_code('bank', self.env.company.id)
-            else:
-                record.new_journal_code = self.linked_journal_id.code
-
     @api.model
     def create(self, vals):
         """ This wizard is only used to setup an account for the current active
         company, so we always inject the corresponding partner when creating
         the model.
         """
         vals['partner_id'] = self.env.company.partner_id.id
+        print(vals)
```

Comment:
Noooooooooo it's the only thing, otherwise it was good

Reply:
You saw nothing